Sources/StreamChat/Models/Attachments/AttachmentTypes.swift

```diff
@@ -238,6 +241,7 @@ public enum AttachmentFileType: String, Codable, Equatable, CaseIterable {
         "text/plain": .txt,
         "audio/mp3": .mp3,
         "audio/mp4": .m4a,
+        "audio/aac": .aac,
```

Comment:
question: If we upload aac on iOS, would Android be able to play it?

Reply:
They should be. I'm going to ask the Android team and and depending on their answer i'll update the format in the next PR.